### ISSUE TITLE
Impossible to add iOS device as a remote in Dash for Mac

### ISSUE THREAD

**Issue by kaiky25** [Kapeli/Dash-iOS#34]
Hi there,

When I try to add a new iOS remote in Dash for Mac the pairing process starts but no inputs are shown in the iOS app to continue the process. Seems like it doesn't even recognize my Mac trying to pair with it, could it be a code signing problem?

**Comment by Kapeli** [comment]
It could. Can you check the console on both the Mac and iOS side for any messages?

Also try to do it on a different network.

**Comment by kaiky25** [comment]
Rebuilt the app on Xcode and checked for any logs on the console when I tried to pair the remote, nothing there tho. After I reinstalled Dash on my Mac and built the iOS app on the local simulator and tried to pair with the Mac app and I was able to do it, then I tried to pair again with my iPhone and finally got it to work. Maybe it was something to do with my .dashsync file, some weird bug as I've paired before with the iOS app before formatting my Mac. Anyhow, all is good now, thanks for your help!
